src/reducers/__fixtures__/trendsAggs.jsx

```diff
@@ -1 +1 @@
83020800000, "doc_count": 7 } ] }, "issue": { "doc_count_error_upper_bound": 0, "sum_other_doc_count": 0, "buckets": [ { "key": "Problem with fees", "doc_count": 17, "trend_period": { "buckets": [ { "key_as_string": "2020-03-01T00:00:00.000Z", "key": 1583020800000, "doc_count": 4 } ] } }, { "key": "Charged too much interest", "doc_count": 6, "trend_period": { "buckets": [ { "key_as_string": "2020-03-01T00:00:00.000Z", "key": 1583020800000, "doc_count": 2 } ] } }, { "key": "Unexpected increase in interest rate", "doc_count": 4, "trend_period": { "buckets": [ { "key_as_string": "2020-03-01T00:00:00.000Z", "key": 1583020800000, "doc_count": 1 } ] } } ] } } ] } }, "min_date": { "value": 1584032400000, "value_as_string": "2020-03-12T12:00:00-05:00" }, "dateRangeArea": { "doc_count": 532, "dateRangeArea": { "buckets": [ { "key_as_string": "2020-03-01T00:00:00.000Z", "key": 1583020800000, "doc_count": 106 }, { "key_as_string": "2020-04-01T00:00:00.000Z", "key": 1585699200000, "doc_count": 374 }, { "key_as_string": "2020-05-01T00:00:00.000Z", "key": 1588291200000, "doc_count": 52 } ] } }, "tags": { "doc_count": 532, "tags": { "doc_count_error_upper_bound": 0, "sum_other_doc_count": 0, "buckets": [ { "key": "Servicemember", "doc_count": 70, "trend_period": { "buckets": [ { "key_as_string": "2020-05-01T00:00:00.000Z", "key": 1588291200000, "doc_count": 7, "interval_diff": { "value": -43 } }, { "key_as_string": "2020-04-01T00:00:00.000Z", "key": 1585699200000, "doc_count": 50, "interval_diff": { "value": 37 } }, { "key_as_string": "2020-03-01T00:00:00.000Z", "key": 1583020800000, "doc_count": 13 } ] } }, { "key": "Older American", "doc_count": 30, "trend_period": { "buckets": [ { "key_as_string": "2020-05-01T00:00:00.000Z", "key": 1588291200000, "doc_count": 1, "interval_diff": { "value": -21 } }, { "key_as_string": "2020-04-01T00:00:00.000Z", "key": 1585699200000, "doc_count": 22, "interval_diff": { "value": 15 } }, { "key_as_string": "2020-03-01T00:00:00.000Z", "key": 1583020800000, "doc_count": 7 } ] } }, { "key": "Older American, Servicemember", "doc_count": 7, "trend_period": { "buckets": [ { "key_as_string": "2020-05-01T00:00:00.000Z", "key": 1588291200000, "doc_count": 1, "interval_diff": { "value": -4 } }, { "key_as_string": "2020-04-01T00:00:00.000Z", "key": 1585699200000, "doc_count": 5, "interval_diff": { "value": 4 } }, { "key_as_string": "2020-03-01T00:00:00.000Z", "key": 1583020800000, "doc_count": 1 } ] } } ] } }, "dateRangeBuckets": { "doc_count": 121054, "dateRangeBuckets": { "buckets": [ { "key_as_string": "2020-01-01T00:00:00.000Z", "key": 1577836800000, "doc_count": 21519 }, { "key_as_string": "2020-02-01T00:00:00.000Z", "key": 1580515200000, "doc_count": 25096 }, { "key_as_string": "2020-03-01T00:00:00.000Z", "key": 1583020800000, "doc_count": 29506 }, { "key_as_string": "2020-04-01T00:00:00.000Z", "key": 1585699200000, "doc_count": 35112 }, { "key_as_string": "2020-05-01T00:00:00.000Z", "key": 1588291200000, "doc_count": 9821 } ] } } }
```

Comment:
Is this parseable as a JSON block?   I had problems trying to run an JSON prettifier on it when I was trying to change the values

Reply:
yes. 

![prettify](https://user-images.githubusercontent.com/1038306/86974913-253ed380-c145-11ea-9cc5-cb1968931730.gif)